storage/src/dynamic_boc_rc_db.rs

```diff
@@ -875,4 +884,77 @@ impl OrderedCellsStorage for OrderedCellsStorageAdapter {
         self.index2.destroy()?;
         Ok(())
     }
+}
+
+
+struct RawCellsCache(Cache<UInt256, Bytes, CellSizeEstimator, ahash::RandomState>);
```

Comment:
Why do you store raw bytes in the cache, not `Arc<StorageCell>`?

Reply:
Raw data contains only 1 cell, without any references. So in memory it will take 80 bytes less then StorageCell. In relation to millions of cells in cache it will make big boost. Deserialization to StorageCell is a cheap operation and is performed fast.